traffic_ops/traffic_ops_golang/physlocation/phys_locations.go

```diff
@@ -20,8 +20,12 @@ package physlocation
  */
 
 import (
+	"database/sql"
+	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/apache/trafficcontrol/lib/go-log"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/util/ims"
```

Comment:
These two imports aren't placed in the correct block.

Reply:
Fixed it